src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -33,7 +33,7 @@
     @APIDoc("NVMe-oF Gateway Management API", "NVMe-oF Gateway")
     class NVMeoFGateway(RESTController):
         @EndpointDoc("Get information about the NVMeoF gateway")
-        @NvmeofCLICommand("nvmeof gw info", model.GatewayInfo)
+        @NvmeofCLICommand("nvmeof gateway info", model.GatewayInfo)
```

Comment:
I suggest that we still keep the old syntax for the following reasons:
* This is a breaking change. Any integration/user/script using the old syntax will stop working.
* In CLI is common to use short-cuts for long tokens (`ceph fs` for `ceph filesystem`, `rm` for `remove`, `svc` for `service`).

So my suggestion is that we keep this AND add the long syntax, for example, as aliases:
  ```suggestion
        @NvmeofCLICommand("nvmeof gateway info", alias="nvmeof gw info", model=model.GatewayInfo)
```

Reply:
I was sure that supporting aliases is a big deal but your comment made it click for me. just implemented it, I hope you will like it :) Thanks!